doc/using_xgboost.rst

```diff
@@ -0,0 +1,212 @@
+#########################################
+Use XGBoost with the SageMaker Python SDK
+#########################################
+
+.. contents::
+
+eXtreme Gradient Boosting (XGBoost) is a popular and efficient machine learning algorithm used for regression and classification tasks on tabular datasets.
+It implements a technique known as gradient boosting on trees, which performs remarkably well in machine learning competitions.
+
+Amazon SageMaker supports two ways to use the XGBoost algorithm:
+
+ * XGBoost built-in algorithm
+ * XGBoost open source algorithm
```

Comment:
maybe "open source library"? (taking inspiration from the first sentence of the [XGBoost docs](https://xgboost.readthedocs.io/en/latest/))

Reply:
I'm going to punt on this for now. I don't think it will confuse anyone as is.